dlpy/network.py

```diff
@@ -0,0 +1,1598 @@
+#!/usr/bin/env python
+# encoding: utf-8
+#
+# Copyright SAS Institute
+#
+#  Licensed under the Apache License, Version 2.0 (the License);
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+''' A Network is way to compose layers: the topological form of a Model '''
+
+import os
+
+from dlpy.layers import Layer
+from dlpy.utils import DLPyError, input_table_check, random_name, check_caslib, caslibify
+from .layers import InputLayer, Conv2d, Pooling, BN, Res, Concat, Dense, OutputLayer, Keypoints, Detection
+import collections
+import pandas as pd
+import warnings
+
+
+class Network(Layer):
+
+    def __init__(self, *args, **kwargs):
```

Comment:
I am not really happy with the init method. it basically accepts anything and everything.